Ginger/GingerCore/ALM/QCRestAPI/QCRestAPIConnect.cs

```diff
@@ -25,6 +25,8 @@ public static class QCRestAPIConnect
 
         public static bool ConnectQCServer(string qcServerUrl, string qcUserName, string qcPassword)
         {
+            if (qcServerUrl.EndsWith("qcbin"))
+                qcServerUrl = qcServerUrl.Remove(qcServerUrl.Length - 5);
             if (QcRestClient == null || qcServerUrl != ServerURL || qcUserName != UserName || qcPassword != Password)
```

Comment:
change to qcServerUrl.ToLower().EndsWith("qcbin"), to not failed on upper case issue.
also please handle Codacy issues